### PR TITLE
Nodejs guides 4–6

### PR DESCRIPTION
**Pull request by shvgn**
Guides

  - 040 (db)
  - 050 (files w/minio)
  - 080 (config and secrets)
 
Unification and inheritance of the codebase

### CODE REVIEW

File: .werf/nginx-dev.conf [hunk 1]
```diff
@@ -40,43 +40,43 @@ http {
...
-        location / {
-          proxy_redirect    off;
-          proxy_set_header  Host              werf.io;
-          proxy_set_header  X-Real-IP         $remote_addr;
-          proxy_set_header  X-Forwarded-For   $proxy_add_x_forwarded_for;
-          proxy_pass https://werf.io;
-        }
-    }
+    # server {
+    #     charset utf-8;
```
Comment (shvgn) [inline .werf/nginx-dev.conf:44]: - [ ] Not to forget to revert

File: docker-compose.yml [hunk 2]
```diff
@@ -21,8 +21,9 @@ services:
       jekyll serve -t --disable-disk-cache --config _config.yml,_config_dev.yml,_config_ru.yml --baseurl /guides --destination /tmp/_site -P 80"
     volumes:
       - "./:/srv/jekyll-data/"
-    depends_on:
-      - en
+    restart: unless-stopped
```
Comment (shvgn) [inline docker-compose.yml:24]: - [ ] Not ot forget to revert

File: _includes/ru/200_real_apps/040_db/nodejs/001_preparing_stateful_app.md.liquid [hunk 3]
```diff
@@ -1 +1 @@
-**{{- "TODO (_includes/ru/200_real_apps/040_db/nodejs/001_preparing_stateful_app.md.liquid)." -}}**
+{{- "добавить бибилиотеки для работы с БД и подготовить для этого приложение" -}}
```
Comment (Zhbert) [inline _includes/ru/200_real_apps/040_db/nodejs/001_preparing_stateful_app.md.liquid:1]:
```suggestion
{{- "добавить библиотеки для работы с БД и подготовить для этого приложение." -}}
```

File: _includes/ru/200_real_apps/040_db/nodejs/005_major_changes.md.liquid [hunk 4]
```diff
@@ -1 +1,21 @@
-1. **TODO (_includes/ru/200_real_apps/040_db/nodejs/005_major_changes.md.liquid): список изменений.**
+1. Установлены зависимости для работы с MySQL
+1. Добавлена конфигцрация доступа к БД
```
Comment (Zhbert) [inline _includes/ru/200_real_apps/040_db/nodejs/005_major_changes.md.liquid:2]:
```suggestion
1. Добавлена конфигурация доступа к БД
```

File: _includes/ru/200_real_apps/040_db/nodejs/005_major_changes.md.liquid [hunk 5]
```diff
@@ -1 +1,21 @@
...
+1. Добавлена модель и миграция схемы БД
+1. Добавлены маршруты, использующие новую модель
+
+Новые зависимости была установлены командой
```
Comment (Zhbert) [inline _includes/ru/200_real_apps/040_db/nodejs/005_major_changes.md.liquid:6]:
```suggestion
Новые зависимости были установлены командой:
```

File: _includes/ru/200_real_apps/040_db/nodejs/020_setup_db_work.md.liquid [hunk 6]
```diff
@@ -1,3 +1,4 @@
-```
-TODO (_includes/ru/200_real_apps/040_db/nodejs/020_setup_db_work.md.liquid): настройка приложения на работу с новой БД.
-```
\ No newline at end of file
+{% include snippetcut_example path="config/database.json" syntax="json" examples=page.examples %}
+
+Sequlize предполагает, что полкючения для всех окружений будут определены в одном файле. Пока что мы
```
Comment (Zhbert) [inline _includes/ru/200_real_apps/040_db/nodejs/020_setup_db_work.md.liquid:3]:
```suggestion
Sequlize предполагает, что подключения для всех окружений будут определены в одном файле. Пока что мы
```

File: _includes/ru/200_real_apps/050_s3/nodejs/001_preparing.md.liquid [hunk 7]
```diff
@@ -1 +1,15 @@
...
+Добавим библиотеки для работы с файлами:
+
+```shell
+npm i @aws-sdk/client-s3 express-fileupload readable-web-to-node-stream
+```
+
+- `@aws-sdk/client-s3` — для работы с хранилищем, совместимым с AWS S3
+- `express-fileupload` — для простого получения файла в запросе express
+- `readable-web-to-node-stream` — для конвертирования потока *web stream* в *node stream*, что
+  пригодится для отправки файлу клиенту
```
Comment (Zhbert) [inline _includes/ru/200_real_apps/050_s3/nodejs/001_preparing.md.liquid:12]:
```suggestion
пригодится для отправки файлов клиенту
```

File: _includes/ru/200_real_apps/080_configuration/nodejs/085_configs_including.md.liquid [hunk 8]
```diff
@@ -1,3 +1,24 @@
+После передачи переменных окружения в контейнер остаётся только подставить их в конфигурационных
+файлах приложения.
+
+Чтобы единообразно управлять конфигурацией приложения, уставновим модуль `config`:
```
Comment (Zhbert) [inline _includes/ru/200_real_apps/080_configuration/nodejs/085_configs_including.md.liquid:4]:
```suggestion
Чтобы единообразно управлять конфигурацией приложения, установим модуль `config`:
```

**Review by Zhbert — CHANGES_REQUESTED** [review]
I found a couple of small typos

**Comment by Zhbert** [comment]
I added layout:wip on page 040_db (we need to do something with the partial hierarchy later, where FIXME is marked)

**Comment by shvgn** [comment]
Fixes applied

**Review by Zhbert — APPROVED** [review]

**Pull request merged**